### ISSUE TITLE
Crash when using pause/play button on a bluetooth device

### ISSUE THREAD

**Issue by ChunkyProgrammer** [TeamNewPipe/NewPipe#6233]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1) Open NewPipe (don't play a video)
2) Click Pause/Play button a few times in a row

### Actual behaviour
App crashes

### Expected behavior
App ignores play/pause button when nothing is in the queue

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ none
* __Version:__ 0.21.2
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'int org.schabi.newpipe.player.playqueue.PlayQueue.getIndex()' on a null object reference
	at org.schabi.newpipe.player.Player.onCompleted(Player.java:2154)
	at org.schabi.newpipe.player.Player.changeState(Player.java:1993)
	at org.schabi.newpipe.player.Player.onPlayerStateChanged(Player.java:1913)
	at com.google.android.exoplayer2.ExoPlayerImpl.lambda$updatePlaybackInfo$13(ExoPlayerImpl.java:1052)
	at com.google.android.exoplayer2.-$$Lambda$ExoPlayerImpl$eAKzukwdd6P91AoYEF8JC1HDnJ4.invoke(Unknown Source:4)
	at com.google.android.exoplayer2.util.ListenerSet$ListenerHolder.invoke(ListenerSet.java:297)
	at com.google.android.exoplayer2.util.ListenerSet.lambda$queueEvent$0(ListenerSet.java:184)
	at com.google.android.exoplayer2.util.-$$Lambda$ListenerSet$NbKDn9xtItiyMgYZmjIx_Sv1FFQ.run(Unknown Source:6)
	at com.google.android.exoplayer2.util.ListenerSet.flushEvents(ListenerSet.java:205)
	at com.google.android.exoplayer2.ExoPlayerImpl.updatePlaybackInfo(ExoPlayerImpl.java:1101)
	at com.google.android.exoplayer2.ExoPlayerImpl.prepare(ExoPlayerImpl.java:333)
	at com.google.android.exoplayer2.SimpleExoPlayer.prepare(SimpleExoPlayer.java:1268)
	at com.google.android.exoplayer2.DefaultControlDispatcher.dispatchPrepare(DefaultControlDispatcher.java:57)
	at com.google.android.exoplayer2.ext.mediasession.MediaSessionConnector$ComponentListener.onPlay(MediaSessionConnector.java:1156)
	at android.support.v4.media.session.MediaSessionCompat$Callback$MediaSessionCallbackApi21.onPlay(MediaSessionCompat.java:1614)
	at android.media.session.MediaSession$Callback.onMediaButtonEvent(MediaSession.java:928)
	at android.support.v4.media.session.MediaSessionCompat$Callback$MediaSessionCallbackApi21.onMediaButtonEvent(MediaSessionCompat.java:1604)
	at android.media.session.MediaSession$CallbackMessageHandler.handleMessage(MediaSession.java:1566)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7867)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:981)

```
</details>
<hr>

### Device info

 - Android version/Custom ROM version:
 - Device model:

**Comment by ChunkyProgrammer** [comment]
I imagine it's some sort of async issue?

**Comment by mhmdanas** [comment]
I'll look into this.

**Comment by skyGtm** [comment]
Looking at the exception 
> Attempt to invoke virtual method 'int org.schabi.newpipe.player.playqueue.PlayQueue.getIndex()

https://github.com/TeamNewPipe/NewPipe/pull/6570 might fix the issue.

**Comment by mhmdanas** [comment]
Should be fixed by #6570.
